share/revng/test/tests/analysis/ImportFromCAnalysis/enum-with-a-broken-annotation/expected-error.txt

```diff
@@ -0,0 +1,7 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: An enum with a non-primitive underlying type.
+import-from-c: An enum with a non-primitive underlying type.
```

Comment:
Also I think that it should be possible from the error message to discriminate between this case (the underlying type is not a valid type) from the case where the underlying type is a valid type but not a valid underlying type for an enum.

Reply:
Done.